src/config.ini.example

```diff
@@ -3,6 +3,7 @@ gps_default = 0, 0
 mysql_path = mysql://root:toor@localhost:3306
 wigle_name = AIC1234
 wigle_key = deadbeef
+ocid_cache_len = 120 ;OpenCellID cell tower cache timeout in minutes.
```

Comment:
I set it as a tunable parameter because I wasn't sure if I was missing something in terms of how long the cache should last. I've got not problem just making that static or an optional thing.

Reply:
I thought I posted a comment here earlier, but I'm not seeing it now. I think removing this or making it optional is fine. I just made it configurable because I wasn't sure what value to use and just sort of guessed.